src/scripts/dbgap/dbgap_classes.py

```diff
@@ -240,20 +232,20 @@ def __init__(self, sample, read_group):
     def get_submitter_id(self):
         pairing_code = self.read_group.pairing_code()
         pdo_or_wr = self.read_group.get_pdo_or_wr()
-        formatted_data_type = self.sample.formatted_data_type()["constant"].replace(" ", "_")
+        formatted_data_type = self.sample.formatted_data_type["constant"].replace(" ", "_")
 
         return f"{self.sample.phs}.{pdo_or_wr}.{self.read_group.library_name}.{pairing_code}.{self.sample.alias}.{self.sample.project}.{formatted_data_type}.{self.sample.version}"
 
     def get_file_name(self):
         return f"{self.get_submitter_id()}.add.experiment.xml"
 
     def get_title(self):
-        repo = self.sample.get_biospecimen_repo()
+        repo = self.sample.biospecimen_repo
```

Comment:
Looks like `repo` could potentially be an empty string. Does the return statement need to get adjusted in the event that it is an empty string?